pegasus/sites.v3/code.org/views/district_partners_list.haml

```diff
@@ -1,12 +1,13 @@
 - district_partners = DB[:cdo_district_partners].all
 
-%table
-  %thead
-    %tr
-      %th=hoc_s(:heading_district)
-      %th=hoc_s(:heading_state)
-  %tbody
-    - district_partners.each do |value|
+.table-wrapper
```

Comment:
The table on this page https://staging.code.org/educate/district/partners uses the same new responsive styles so I updated this too.